rskj-core/src/test/java/org/ethereum/net/server/ChannelManagerImplTest.java

```diff
@@ -70,10 +73,43 @@ void getNumberOfPeersToSendStatusTo() {
 
     @Test
     void blockAddressIsAvailable() throws UnknownHostException {
-        ChannelManagerImpl channelManagerImpl = new ChannelManagerImpl(new TestSystemProperties(), null);;
+        ChannelManagerImpl channelManagerImpl = new ChannelManagerImpl(new TestSystemProperties(), null);
         Assertions.assertTrue(channelManagerImpl.isAddressBlockAvailable(InetAddress.getLocalHost()));
     }
 
+    @Test
+    void blockAddressCheck_logInvalidInetAddressException() throws UnknownHostException {
+        TestSystemProperties config = mock(TestSystemProperties.class);
+        when(config.maxConnectionsAllowed()).thenReturn(1);
+        when(config.networkCIDR()).thenReturn(0);
+
+        SyncPool syncPool = mock(SyncPool.class);
+        ChannelManagerImpl channelManager = new ChannelManagerImpl(config, syncPool);
+
+        String remoteId = "remoteId";
+        NodeManager nodeManager = new NodeManager(null, config);
+
+        Channel peer = spy(new Channel(null, null, nodeManager, null, null, null, remoteId));
+        peer.setInetSocketAddress(new InetSocketAddress("127.0.0.1",5554));
+        peer.setNode(new NodeID(TestUtils.generatePeerId("peer")).getID());
+        when(peer.isProtocolsInitialized()).thenReturn(true);
+        when(peer.isActive()).thenReturn(true);
+        when(peer.isUsingNewProtocol()).thenReturn(true);
+
+        Channel otherPeer = new Channel(null, null, nodeManager, null, null, null, remoteId);
+        otherPeer.setInetSocketAddress(new InetSocketAddress("127.0.0.1",5554));
+
+        channelManager.add(peer);
+        channelManager.tryProcessNewPeers();
+
+        Logger logger = mock(Logger.class);
+        TestUtils.setFinalStatic(channelManager, "logger", logger);
+
+        channelManager.isAddressBlockAvailable(otherPeer.getInetSocketAddress().getAddress());
```

Comment:
Maybe you should check that in this scenario this method returns `true`.

Reply:
Done